.env

```diff
@@ -65,3 +65,4 @@ LOG_LEVEL_ROOT=TRACE
 LOG_LEVEL_MONGODB=TRACE
 
 BUILD_CACHE_REPO=openfoodfacts/openfoodfacts-build-cache
+CPANMOPTS=--with-develop --with-feature=off_server_dev_tools
```

Comment:
This should not be included in the PR

Reply:
@himanshisrestha if you use linux or mac, you might use [direnv](https://openfoodfacts.github.io/openfoodfacts-server/dev/how-to-use-direnv/)